assets/js/hooks/session.js

```diff
@@ -988,6 +993,25 @@ const Session = {
       }
     }
   },
+  customViewSelection() {
+    settingsStore.getAndSubscribe((settings) => {
```

Comment:
We also need to unsubscribe when custom view is disabled. We can make it such that `getAndSubscribe` returns an unsubscribe function (similarly to `pub_sub.js`). Then we can store it in `this.unsubscribeCustomViewFromSettings` and call when the view is disabled.

Reply:
Note: this is still relevant, we need to unsubscribe from the store :)

Reply:
oh, I missed this comment, I'll take a look and resolve it. Thanks